prow/config/templates/plugins-ConfigMap.yaml

```diff
@@ -46,6 +46,7 @@ data:
         - label
         - lgtm
         - lifecycle
+        - require-matching-label
```

Comment:
To keep to scope of this PR simple and precise. Can we make this in a separate PR? We'll need other plugins after #263 is merged too.

Reply:
Done